src/setfit/data.py

```diff
@@ -1,8 +1,8 @@
-from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Dict, List, Tuple, Union
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, Union
```
Required for recommended changes below: